### PR TITLE
Title-Component-Changes-with-styles-and-policies (https://becurious.edcast.eu/journey/aem-demo-website/cards/109701811)

### PR DESCRIPTION
**Pull request by ashankarprakash**

### CODE REVIEW

**Review by azouaoui-deloitte — CHANGES_REQUESTED** [review]
There is a new conf folder not in the right place,
The policies are not persisted in the right place in the repository, when I build the project I don't see your changes,
You need to put your policies under ui.content/src/main/content/jcr_root/conf/aemaacsdemowebsite/settings/wcm/policies/.content.xml

File: conf/aemaacsdemowebsite/settings/wcm/policies/.content.xml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:rep="internal" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:sling="http://sling.apache.org/jcr/sling/1.0"
+    jcr:mixinTypes="[rep:AccessControllable]"
+    jcr:primaryType="cq:Page">
+    <rep:policy/>
+    <aemaacsdemowebsite jcr:primaryType="nt:unstructured">
+        <components jcr:primaryType="nt:unstructured">
+            <form/>
+            <languagenavigation/>
+            <image/>
+            <text/>
```
Comment (azouaoui-deloitte) [inline conf/aemaacsdemowebsite/settings/wcm/policies/.content.xml:11]: this is the result of a copy paste from a package coming from CRX, you need to commit only the changes (policies) you need

File: ui.frontend/src/main/webpack/components/_title.scss [hunk 2]
```diff
@@ -1,3 +1,42 @@
...
+.headline-four{
+    font-size: 32px;
+}
+.headline-five{
+    font-size: 24px;
+}
+.headline-six{
+    font-size: 18px;
+}
+@media only screen and (max-width: 767px) {
```
Comment (manpkaur9) [inline ui.frontend/src/main/webpack/components/_title.scss:23]: Variable can be used here

File: ui.frontend/src/main/webpack/components/_title.scss [hunk 3]
```diff
@@ -1,3 +1,42 @@
...
+    .headline-one{
```
Comment (manpkaur9) [inline ui.frontend/src/main/webpack/components/_title.scss:24]: also can be handled within line 5 block

Comment (manpkaur9) [reply ui.frontend/src/main/webpack/components/_title.scss:24]: .headline-one{
    font-size: 60px;
    @media only screen and (max-width: 767px) {
          font-size: 48px;
    }
}

File: conf/aemaacsdemowebsite/settings/wcm/policies/.content.xml (hunk 1 above)
Comment (ashankarprakash) [reply conf/aemaacsdemowebsite/settings/wcm/policies/.content.xml]: Changes made in the recent commit.

File: ui.frontend/src/main/webpack/components/_title.scss (hunk 2 above)
Comment (ashankarprakash) [reply ui.frontend/src/main/webpack/components/_title.scss]: Changes made in the recent commit.

File: ui.frontend/src/main/webpack/components/_title.scss (hunk 3 above)
Comment (ashankarprakash) [reply ui.frontend/src/main/webpack/components/_title.scss]: Changes made in the recent commit.

File: ui.content/src/main/content/jcr_root/conf/aemaacsdemowebsite/settings/wcm/policies/.content.xml [hunk 4]
```diff
@@ -176,14 +176,84 @@
                     <jcr:content jcr:primaryType="nt:unstructured"/>
                 </policy_641475696923109>
                 <policy_641528232375303
+                    cq:styleDefaultClasses="heading-sizes"
```
Comment (azouaoui-deloitte) [inline ui.content/src/main/content/jcr_root/conf/aemaacsdemowebsite/settings/wcm/policies/.content.xml:179]: Why did we add this default class? is it used somewhere?

**Review by azouaoui-deloitte — APPROVED** [review]

**Pull request merged**